ckanext/dcatapchharvest/tests/test_swiss_dcatap_profile_parse.py

```diff
@@ -149,13 +145,9 @@ def test_dataset_issued_with_year_before_1900(self):
         dataset = datasets[0]
 
         # Check date values
-        eq_(dataset['issued'], -2398377600)
-        issued = datetime.fromtimestamp(dataset['issued'])
-        eq_(issued.date().isoformat(), u'1893-12-31')
+        eq_(dataset['issued'], u'1893-12-31T00:00:00')
 
-        eq_(dataset['modified'], 1524528000)
-        modified = datetime.fromtimestamp(dataset['modified'])
-        eq_(modified.date().isoformat(), u'2018-04-24')
```

Comment:
Why do you delete all these checks? 
don't we want to check what is happening, when we get not only isoformat, but also date in Unix timestampp?

Reply:
We shouldn't get unix timestamps: this is part of DCAT-AP CH, see also the handbook: https://handbook.opendata.swiss/de/content/glossar/bibliothek/dcat-ap-ch.html#dcat-dataset-issued 

Dates must be provided as isodates for all DCAT properties. See also here for DCAT: https://www.w3.org/TR/vocab-dcat-3/#Property:resource_release_date

Reply:
then it is fine by me

Reply:
I see that we shouldn't get anything else that what is tested against here, but the idea of testing is also to check if (for any reason) nonsense is coming in. I would still like to see that we make sure, that we deal with it properly, in case nonsense data comes in.

Reply:
Ich stimme nicht damit überein, einen Test zu lassen, der aus meiner Sicht unsinnig ist. Wir testen hier gegen Fixtures.Es gibt bei diesen Tests nur einen test der die komplette Übenrahme eines Daasets testet. Dieser Test, den ich angepasst habe, testet den Okay Fall. Deshalb musste der Test auf das erwartete Format umgestellt werden. Man kann gerne einen neuen Test hinzufügen, der den NOK Fall testet. Solche Tests gibt es ja auch bereits. Mache bitte auch hier einen konktreten Vorschlag welchen neune Test-Fall ich hinzufügen soll.